### PR TITLE
Add initial documentation  (#119)

### PR DESCRIPTION
**Pull request by jonaprieto**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -69,5 +69,9 @@ UPDATES-FOR-CHANGELOG.org
 
 # C Code generation
 *.wasm
+<<<<<<< HEAD
```
Comment (paulcadman) [inline .gitignore:72]: looks like a merge conflict needs fixing here?

**Comment by paulcadman** [comment]
The CI has published the documentation site, but I can't find the URL. I tried https://heliaxdev.github.io/minijuvix/ but it's 404. What's the expected URL of the GitHub pages deployment?

**Comment by jonaprieto** [comment]
> I tried https://heliaxdev.github.io/minijuvix/

This is correct. There is a problem with the action and the published folder: an issue coping the HTML files with `cp`.  I'm investigating it.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -182,7 +182,8 @@ jobs:
 
   docs:
     runs-on: ubuntu-latest
-    if:  ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
+    # Enable before merging.
+    # if:  ${{ github.ref == 'refs/heads/master' }}
```
[the remark this answers is not in the document]
Comment (jonaprieto) [reply .github/workflows/ci.yml]: Done. ;)

**Pull request merged**